### ISSUE TITLE
Define damage types

### ISSUE THREAD

**Issue by fshpsmgc** [aldopain/cyberpunk#52]
Нужна ли нам дифференциация типов урона?

**Comment by danibord** [comment]
Из этого же еще вытекает дифференциация типов защиты?

**Comment by danibord** [comment]
По хорошему, мне кажется это добавит кучу простора для всяких скиллов/ апргейдов, но вот тоже вопрос нужно ли так переусложнять систему

**Comment by aldopain** [comment]
Вообще я предлагаю сделать пока в зачаточном варианте. Тип это даст много возможностей, но при этом ее можно быстро обрезать, если нам не понравится. 
Пока сделать разделение кинетический/рубящий/ментальный/огонь, например, и забить. Потом сможем расширить.

**Comment by aldopain** [comment]
Да, вытекают типы защиты. На героя вешать необязательно, но можно на врагов, тип эти устойчивы к огню, а те к рубящему. Делается легко.

**Comment by aldopain** [comment]
Алсо, есть веселый факап. 
Нам нужна более-менее адекватная система хп. Сейчас у нас там int и система защиты будет конфликтовать/работать херово, если не будет условно 100 хп у персонажа.
